### ISSUE TITLE
Explanation about different loss functions

### ISSUE THREAD

**Issue by hhsecond** [udacity/dlnd-issue-reports#155]
Descriptions given in the sessions was not enough to convince the students, why cross entropy should be used over softmax for categorical output

**Comment by bdelhaisse** [comment]
It seems that you are confusing loss functions (such as MSE, cross-entropy,...) and activation layers (such as sigmoid, softmax,...). The cross-entropy loss is used together with the softmax.
Anyway, they should maybe add a note to explicitly differentiate between these 2 concepts as you are probably not the only one mixing with all these different definitions. Maybe, a summary section resuming the important points could also be added at the end of each lesson.

**Comment by Brok-Bucholtz** [comment]
This is something we want to cover along with a list of over stuff.  We'll add this to our backlog.
